components/download-info.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div :class="[classes, 'container']">
+  <div class="container">
```

Comment:
Remove the dark mode from the download modal because not all the buttons adjust and it looks pretty bad